src/main/java/core/basesyntax/WithArea.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
+
+public interface WithArea {
```

Comment:
Let's give a more informative name of the interface, for example, AreaCalculator